profiles/util_test.py

```diff
@@ -145,3 +150,54 @@ def test_make_thumbnail(self):
             mocked.assert_called_with(width, height, thumbnail_size)
             assert thumb_image.width == thumb_width
             assert thumb_image.height == thumb_height
+
+
+@pytest.mark.django_db
```

Comment:
was getting some exception on access test db in these tests and in exception it suggest me to use this.